src/main/java/io/strimzi/kafka/bridge/KafkaBridgeAdmin.java

```diff
@@ -84,6 +87,31 @@ public CompletionStage<Set<String>> listTopics() {
         return promise;
     }
 
+    /**
+     * Creates a topic with given name
```

Comment:
It would be helpful to complete the description mentioning also optional partitions and replication factor.